mobile-widgets/qmlmanager.h

```diff
@@ -58,6 +58,8 @@ class QMLManager : public QObject {
 	};
 
 	static QMLManager *instance();
+	void registerError(const QString &error);
```

Comment:
should be declared `Q_INVOKABLE void registerError...` if `invokeMethod()` is used.